next/components/forms/segments/CorrespondenceAddressModal/CorrespondenceAddressModal.tsx

```diff
@@ -0,0 +1,41 @@
+import CorrespondenceAddressForm, {
+  CorrespondenceAddressData,
+} from 'components/forms/segments/CorrespondenceAddressForm/CorrespondenceAddressForm'
+import Modal from 'components/forms/widget-components/Modals/Modal'
+import { useTranslation } from 'next-i18next'
+import { useState } from 'react'
+
+interface Props {
+  show: boolean
+  onClose: () => void
+}
+
+const CorrespondenceAddressModal = ({ show, onClose }: Props) => {
+  const { t } = useTranslation('forms')
+  const [errorMessage, setErrorMessage] = useState('')
+
+  const onSubmit = ({ data }: { data?: CorrespondenceAddressData }) => {
+    console.log(data)
```

Comment:
I think you can update in cognito right as a part of this - or is there something blocking such updates ?

or since this is now more on the level of just being a visual component, maybe just move the onSubmit to the parameter and supply the cognito onSubmit wherever we'll actually use this ?